orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineStage.java

```diff
@@ -71,6 +72,17 @@ public void taskGraph(@Nonnull StageExecution stage, @Nonnull TaskNode.Builder b
     }
   }
 
+  @Override
+  public void afterStages(@Nonnull StageExecution parent, @Nonnull StageGraphBuilder graph) {
+    if (parent
```

Comment:
Nitpick: to make the `if` statement easier to understand, what do you think about extracting the condition in a dedicated method?
Something like:


```java
  @Override
  public void afterStages(@Nonnull StageExecution parent, @Nonnull StageGraphBuilder graph) {
    if (shouldSkipDownstreamOutput(parent)) {
      parent.setOutputs(emptyMap());
    }
  }

  private void shouldSkipDownstreamOutput(StageExecution parent) {
    return parent
        .getContext()
        .getOrDefault("skipDownstreamOutput", "false")
        .toString()
        .equals("true")
  }
```

Reply:
done